docs/extensions/services/working-with-users.md

```diff
@@ -0,0 +1,104 @@
+---
+description: Learn about the required services when working with users and granting them access control.
+contributors: Esther Agbaje
+---
+
+# Working with Users
+
+Directus provides a comprehensive set of services that facilitate seamless collaboration with users while enforcing
+access control to specific roles. They include:
+
+- `UsersService`: Extends the `ItemsService` and is used to manage user accounts and perform CRUD operations on user
+  profiles
+- `RolesServices`: Extends the `ItemsService` and enables you to assign roles to users.
+- `PermissionsServices`: Extends the `ItemsService` and allows you to manage the permissions associated with each role.
+
```

Comment:
Create a user?

Reply:
You cover it below as part of create user and assign role, but I think it should be standalone too.